apps/microtvm/ethosu/Makefile_alif.mk

```diff
@@ -0,0 +1,132 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+.PHONY: clean
+
+CC := arm-none-eabi-gcc
+CXX := arm-none-eabi-g++
+AR := arm-none-eabi-ar
+OBJCOPY := arm-none-eabi-objcopy
+
+PWD := $(abspath $(shell pwd))
+BUILD_DIR := $(PWD)/build
+PLATFORM ?= RTSS_HP
```

Comment:
Maybe add information in the README that we can choose between RTSS_HP and RTSS_HE